storefront-reference-architecture-master/cartridges/app_storefront_PD/cartridge/controllers/Home.js

```diff
@@ -1,12 +1,38 @@
-const server = require("server");
-const HookMgr = require("dw/system/HookMgr");
+var server = require("server");
+var cache = require("*/cartridge/scripts/middleware/cache");
+var consentTracking = require("*/cartridge/scripts/middleware/consentTracking");
+var pageMetaData = require("*/cartridge/scripts/middleware/pageMetaData");
 const page = module.superModule;
 server.extend(page);
 
-server.replace("Show", (req, res, next) => {
-    let viewData = res.getViewData();
-    viewData.title = "SFRA";
-    res.setViewData(viewData);
-    next();
-});
+server.replace(
+    "Show",
+    consentTracking.consent,
+    cache.applyDefaultCache,
+    function (req, res, next) {
+        var Site = require("dw/system/Site");
+        var PageMgr = require("dw/experience/PageMgr");
+        var pageMetaHelper = require("*/cartridge/scripts/helpers/pageMetaHelper");
+        var Resource = require("dw/web/Resource");
+
+        var pageDesignerId =
+            Site.current.preferences.custom.pageDesignerHomepage;
```

Comment:
Need to add check if attribute in object as the other developers won't have that meta and the code will fail on their SBs

Reply:
Fixed